modules/private-clusters-about-azure.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * installing/
+
+[id="private-clusters-about-azure_{context}"]
+= Private clusters in Azure
+
+To create a private cluster on Microsoft Azure, you must provide an existing private VNet and subnets to host the cluster. The installation program must also be able to resolve the DNS records that the cluster requires. The installation program configures the Ingress Operator and API server for only internal traffic.
+
+The cluster still requires access to Internet to access the Azure APIs.
+
+The following items are not required or created when you install a private cluster:
+
+* A `BaseDomainResourceGroup`, since the cluster does not create public records
+* Public IP addresses
+* Public DNS records
+* Public load balancers
```

Comment:
This isn't exactly true. Ideally we want this, but because of limitations in Azure, we do create a public load balancer with a dummy rule. Azure does not allow outbound traffic without this, unfortunately.

Reply:
Can you tell me a little more about the dummy rule?